File: patterns/c-patterns/pattern102.c
// Pattern 102: ...

// *********
//  *     *
//   *   *
//    * *
//     *

#include <stdio.h>

int main()
{
    int n = 5;
    int i, j;
    int x = 1, y = 2 * n - 1;
    for (i = 1; i <= n; i++)
    {
        for (j = 1; j < 2 * n; j++)
        {
            if (j == x || j == y || i == 1)
                printf("*");
            else
                printf(" ");
        }
        printf("\n");
        x++;
        y--;
    }
    return 0;
}

File: patterns/javascript-patterns/pattern102.js
// Pattern 102: ...

// *********
//  *     *
//   *   *
//    * *
//     *

let n = 5;
let x = 1;
let y = 2 * n - 1;

for (let i = 1; i <= n; i++) {
    let row = '';

    for (let j = 1; j < 2 * n; j++) {
        if (j === x || j === y || i === 1) {
            row += '*';
        } else {
            row += ' ';
        }
    }

    console.log(row);
    x++;
    y--;
}